src/plugins/intel_gna/src/scale_factor_helper.hpp

```diff
@@ -19,9 +19,8 @@ class Config;
  */
 namespace helpers {
 
-void ApplyInputScaleFactors(const Config& config,
-                            const header_latest::ModelHeader& header,
-                            GnaInputs& inputs);
+void ApplyInputScaleFactors(GnaInputs& inputs, const Config& config, const header_latest::ModelHeader& header);
+void ApplyInputScaleFactors(GnaInputs& inputs, const Config& config);
```

Comment:
just wonder if we need to expose this method in header if it should not be used anywhere beside internally in source file by this method
```
void ApplyInputScaleFactors(GnaInputs& inputs, const Config& config, const header_latest::ModelHeader& header);
```